### PR TITLE
BTA-1691 TZ0 Update

### PR DESCRIPTION
**Pull request by sztupy**

### CODE REVIEW

File: example/build.gradle [hunk 1]
```diff
@@ -3,8 +3,8 @@ plugins {
     id 'application'
 }
 
-group 'co.bitpesa.test'
-version '0.2.0-SNAPSHOT'
+group 'com.transferzero.test'
+version '1.0.0'
```
Comment (emanuelegorga) [inline example/build.gradle:7]: I've noticed that the word **SNAPSHOT** has been removed in some places and not in others - just double checking if it has been made on purpose or not.

Comment (sztupy) [reply example/build.gradle:7]: Yes, it was removed from the examples as they will refer to the published version. It was kept for the main project

Comment (sztupy) [reply example/build.gradle:7]: Good catch btw!

Comment (emanuelegorga) [reply example/build.gradle:7]: Oookay!

**Review by emanuelegorga — APPROVED** [review]
All changes look good to me

**Pull request closed without merging**